### ISSUE TITLE
Need the newest version for Flutter SDK

### ISSUE THREAD

**Issue by MarvinQuevedo** [tangem/tangem-sdk-android-old#338]
Hi, We has problems with the Tangem SDK version in the Flutter Tangem SDK.
for the moment I solved It in iOS using the github repository equals to the Tangem App Pod File.

But in Android I can´t found the way to use your actual version, how I can use
`tangemCardSdk = "release-app_5.4-315"`
in the build.gradle of Flutter Tangem SDK?.

The actual Flutter Tangem SDK build.gradle code

`group 'com.tangem.tangem_sdk'
version '1.0-SNAPSHOT'

buildscript {
    ext.kotlin_version = '1.8.0'
    repositories {
        google()
        mavenCentral()
    }

    dependencies {
        classpath 'com.android.tools.build:gradle:7.1.2'
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

allprojects {
    repositories {
        google()
        mavenCentral()
        maven {url 'https://jitpack.io'}
    }
}

apply plugin: 'com.android.library'
apply plugin: 'kotlin-android'
apply plugin: 'kotlin-kapt'

android {
    compileSdkVersion 33

    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }

    kotlinOptions {
        jvmTarget = '1.8'
    }

    sourceSets {
        main.java.srcDirs += 'src/main/kotlin'
    }

    defaultConfig {
        minSdkVersion 21
    }
}

dependencies {
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"

    implementation "com.github.tangem.tangem-sdk-android:android:3.8.2"
    implementation "com.github.tangem.tangem-sdk-android:core:3.8.2"

    implementation 'com.squareup.moshi:moshi:1.12.0'
    implementation "com.squareup.moshi:moshi-kotlin:1.12.0"
    kapt("com.squareup.moshi:moshi-kotlin-codegen:1.12.0")

    implementation 'androidx.appcompat:appcompat:1.6.1'
    implementation 'androidx.core:core-ktx:1.10.1'
}
`

Here you can see the issue:

https://github.com/tangem/tangem-sdk-flutter/issues/20

**Comment by kozarezvlad** [comment]
We'll publish last sdk release version soon.
For development aims you can use `tangemCardSdk = "release-app_5.4-315"`.
To use it, add this code to "repositories" block in build file:
```
maven {
            url = uri("https://maven.pkg.github.com/tangem/blockchain-sdk-kotlin")
            credentials {
                username = properties.getProperty("gpr.user")
                password = properties.getProperty("gpr.key")
            }
        }
```

Add "gpr.user" and "gpr.key" to `local.properties` file.
"gpr.user" - github user, "gpr.key" - github access token that has read permission. (Guide here https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)

**Comment by kozarezvlad** [comment]
We have released new sdk version 3.9.1, it available in jitpack.

**Comment by MarvinQuevedo** [comment]
How many time Do you think that can take update the flutter SDK? 

I just upgrade the iOS, but for Android, when I upgraded the android libraries, the current flutter android code have code issues by changes in the constructor of the WalletSDK

https://github.com/tangem/tangem-sdk-flutter
